R/FilterStateLogical.R

```diff
@@ -216,6 +216,7 @@ LogicalFilterState <- R6::R6Class( # nolint
           # this observer is needed in the situation when private$selected has been
           # changed directly by the api - then it's needed to rerender UI element
           # to show relevant values
+          finite_values <- reactive(Filter(Negate(is.na), private$x_reactive()))
```

Comment:
finite_values is confusing here non_missing_values is better

Reply:
done